tox.ini

```diff
@@ -132,6 +136,7 @@ install_command =
     pip install {opts} {packages} --isolated --upgrade --force-reinstall
 deps =
     -r{toxinidir}/test-requirements.txt
+	requests[security]
```

Comment:
testenv already has these, why not remove this and let it inherit from there?

Reply:
done